docs/conf/prague/2018/sponsors/prospectus.rst

```diff
@@ -161,52 +172,43 @@ The **Keystone** package costs **€12,000**.
 Other Sponsorship Opportunities
 ===============================
 
-The following à la carte offerings are available either independently or
+The following a la carte offerings are available either independently or
 combined with one of the previous packages to increase visibility at the event.
 
-Financial Assistance
---------------------
+Job Fair
+--------
 
-Write the Docs is always trying to make our community more diverse and
-inclusive. Talk to us about sponsoring travel and/or registration expenses for
-an attendee who wouldn’t otherwise be able to attend.
+Our job fair is a wonderful place to connect with the talented Write the Docs attendees.
+We'll have over 400 people in attendance for our conference,
```

Comment:
worth variable-ing the number of attendees, in case we do this at all the confs?